scripts/nvbench_compare.py

```diff
@@ -250,12 +243,56 @@ def compare_benches(ref_benches, cmp_benches):
             print("")
 
 
-compare_benches(ref_root["benchmarks"], cmp_root["benchmarks"])
+def main():
+
+    help_text = "%(prog)s [reference.json compare.json | reference_dir/ compare_dir/]"
+    parser = argparse.ArgumentParser(prog='nvbench_compare', usage=help_text)
+    parser.add_argument('--threshold-diff',type=float, dest='threshold', default=0.0,
+                        help='only show benchmarks where percentage diff is >= THRESHOLD')
+
+    if len(sys.argv) < 3:
+        parser.print_help()
+        sys.exit(1)
+
+    args = parser.parse_args(sys.argv[3:])
```

Comment:
Does this mean that `--threshold-diff` must be last? Would it be easy to redo this so that

```
nvbench_compare --threshold-diff 5 ref.json comp.json
```

would work too?

Reply:
Updated to allow that